packages/ember/tailwind-webpack/__snapshots__/index.test.ts.snap

```diff
@@ -0,0 +1,127 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`tailwind-webpack > applying to an ember app > works via CLI 1`] = `
+" config/postcss.config.js (new)  | 8 ++++++++
+ config/tailwind.config.js (new) | 9 +++++++++
+ package.json                    | 4 ++++
+ 3 files changed, 21 insertions(+)"
+`;
+
+exports[`tailwind-webpack > applying to an ember app > works via CLI 2`] = `
+"diff --git a/config/postcss.config.js b/config/postcss.config.js
+new file mode 100644
+index 0000000..26885da
+--- /dev/null
++++ b/config/postcss.config.js
+@@ -0,0 +1,8 @@
++module.exports = {
++  plugins: {
++    tailwindcss: {
++      config: 'app/config/tailwind.config.js',
++    },
++    autoprefixer: {},
++  },
++}
+diff --git a/config/tailwind.config.js b/config/tailwind.config.js
+new file mode 100644
+index 0000000..68db9aa
+--- /dev/null
++++ b/config/tailwind.config.js
+@@ -0,0 +1,9 @@
++/** @type {import('tailwindcss').Config} */
++module.exports = {
++  content: ['./app/**/*.{gjs,gts,hbs,html,js,ts}'],
```

Comment:
is this relative to the cwd? seeing `./app` inside `./app/config` makes me do a double-take.

Would it make sense to have something a smidge more robust in here? like:

```js
const path = require('path');

const appRoot = path.join(__dirname, '../../');
const libraries = [/* someLibraryName */ ];

const libraryPaths = libraries
  .map(name => path.dirname(require.resolve(name)));


module.exports = {
  content: [
    `${appRoot}/app/**/*.{js,ts,hbs,gjs,gts,html}`,
    libraryPaths.map(libraryPath => `${libraryPath}/**/*.{js,ts,hbs,gjs,gts,html}`),
  ].flat(),
```

:shrug: 

idk, I've seen folks ask about how to do this (maybe), like here -- https://www.reddit.com/r/emberjs/comments/17r09g2/tailwindcss_in_embroideraddonblueprint_v2/ 
but it's been a while since before that post

Reply:
Tested in my sandbox app and it actually works. Seems like for tailwind the cwd is actually app root? Irrespective to where the config is?

Reply:
Ok added your suggestion. Seems to be working for me. Personal preference, but I replaced the `flat()` call with `...` spread operator.